OWML.ModHelper/Logging/ModConsole.cs

```diff
@@ -26,15 +23,10 @@ protected ModConsole(IOwmlConfig config, IModLogger logger, IModManifest manifes
             Manifest = manifest;
             OwmlConfig = config;
 
-            if (manifest.Name == "OWML")
+            if (Manifest.Name == Constants.OwmlTitle)
             {
-                Instance = this;
+                OwmlConsole = this;
             }
         }
-
-        internal static void CallWriteCallback(IModManifest manifest, string text)
-        {
-            OnConsole?.Invoke(manifest, text);
```

Comment:
uh, where has this gone...?

Reply:
would break backwards compatibility

Reply:
Pretty sure that's intended? 😄

Reply:
Yup, the static event is what stopped us from migrating logging/console stuff to new project. It's intended :)

Reply:
it's `internal` anyway, therefore should not break compatibility